src/Agent.Worker/Telemetry/TelemetryCommandExtension.cs

```diff
@@ -29,13 +35,25 @@ public sealed class PublishTelemetryCommand : IWorkerCommand
         public string Name => "publish";
         public List<string> Aliases => null;
 
+        public bool IsAgentTelemetry;
```

Comment:
Could be done as read-only property with default value of `false`

Reply:
Fixed, thanks!